pandas/plotting/_misc.py

```diff
@@ -280,7 +280,8 @@ def bootstrap_plot(series, fig=None, size=50, samples=500, **kwds):
     samples : int, default 500
         Number of times the bootstrap procedure is performed.
     **kwds
-        Options to pass to matplotlib plotting method.
+        Optional, two dimensional properties from
```

Comment:
This isn't just for matplotlib right? Rather whichever backend is chosen?

If so I think confusing as is, so should make more generic